2015/06/05 22:52:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/prueba/user
---
2015/06/05 23:08:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/prueba/user
---
2015/06/05 23:08:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/prueba/user
---
2015/07/18 17:29:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "postulante".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('postulante')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('postulante')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/postulante...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii/prueba/site/postulante/index
---
2015/07/18 17:31:35 [error] [php] include(USER.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:432)
Stack trace:
#0 C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php(29): spl_autoload_call()
#1 C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#2 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#4 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#5 C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php(94): LoginForm->validate()
#6 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#13 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/site/login
in C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php (29)
in C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php (94)
2015/07/18 17:32:46 [error] [php] include(USER.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:432)
Stack trace:
#0 C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php(29): spl_autoload_call()
#1 C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#2 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#4 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#5 C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php(94): LoginForm->validate()
#6 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#13 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/site/login
in C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php (29)
in C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php (94)
2015/07/18 17:35:30 [error] [php] include(USER.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:432)
Stack trace:
#0 C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php(29): spl_autoload_call()
#1 C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#2 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#4 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#5 C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php(94): LoginForm->validate()
#6 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#13 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/site/login
in C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php (29)
in C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php (94)
2015/07/18 18:04:11 [error] [system.db.CDbCommand] CDbCommand::() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '='21645323' ANDContrasena='Holawe' at line 1. The SQL statement executed was: SELECT Cedula, Contrasena FROM PostulanteWHERE Cedula='21645323' ANDContrasena='Holawe.
in C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php (26)
in C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php (94)
2015/07/18 18:04:11 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '='21645323' ANDContrasena='Holawe' at line 1. The SQL statement executed was: SELECT Cedula, Contrasena FROM PostulanteWHERE Cedula='21645323' ANDContrasena='Holawe' in C:\xampp\htdocs\yii\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\CDbCommand.php(377): CDbCommand->queryInternal('', 0, Array)
#1 C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php(26): CDbCommand->query()
#2 C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#14 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yii/prueba/site/login
HTTP_REFERER=http://localhost:7777/yii/prueba/site/login
---
2015/07/18 18:05:50 [error] [system.db.CDbCommand] CDbCommand::() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '='21645323' ANDContrasena='gfg' at line 1. The SQL statement executed was: SELECT Cedula, Contrasena FROM PostulanteWHERE Cedula='21645323' ANDContrasena='gfg.
in C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php (26)
in C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php (94)
2015/07/18 18:05:50 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '='21645323' ANDContrasena='gfg' at line 1. The SQL statement executed was: SELECT Cedula, Contrasena FROM PostulanteWHERE Cedula='21645323' ANDContrasena='gfg' in C:\xampp\htdocs\yii\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\CDbCommand.php(377): CDbCommand->queryInternal('', 0, Array)
#1 C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php(26): CDbCommand->query()
#2 C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#14 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yii/prueba/site/login
HTTP_REFERER=http://localhost:7777/yii/prueba/site/login
---
2015/07/18 18:07:40 [error] [system.db.CDbCommand] CDbCommand::() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ANDContrasena='Holawe'' at line 1. The SQL statement executed was: SELECT Cedula, Contrasena FROM Postulante WHERE Cedula='21645323' ANDContrasena='Holawe'.
in C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php (26)
in C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php (94)
2015/07/18 18:07:40 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ANDContrasena='Holawe'' at line 1. The SQL statement executed was: SELECT Cedula, Contrasena FROM Postulante WHERE Cedula='21645323' ANDContrasena='Holawe'' in C:\xampp\htdocs\yii\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\CDbCommand.php(377): CDbCommand->queryInternal('', 0, Array)
#1 C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php(26): CDbCommand->query()
#2 C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#14 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yii/prueba/site/login
HTTP_REFERER=http://localhost:7777/yii/prueba/site/login
---
2015/07/18 18:08:36 [error] [system.db.CDbCommand] CDbCommand::() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ANDContrasena='Holawe'' at line 1. The SQL statement executed was: SELECT Cedula, Contrasena FROM postulante WHERE Cedula='21645323' ANDContrasena='Holawe'.
in C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php (26)
in C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php (94)
2015/07/18 18:08:36 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ANDContrasena='Holawe'' at line 1. The SQL statement executed was: SELECT Cedula, Contrasena FROM postulante WHERE Cedula='21645323' ANDContrasena='Holawe'' in C:\xampp\htdocs\yii\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\CDbCommand.php(377): CDbCommand->queryInternal('', 0, Array)
#1 C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php(26): CDbCommand->query()
#2 C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#14 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yii/prueba/site/login
HTTP_REFERER=http://localhost:7777/yii/prueba/site/login
---
2015/07/18 18:09:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "postulante/create".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('postulante/crea...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/prueba/postulante/create
HTTP_REFERER=http://localhost:7777/yii/prueba/site/login
---
2015/07/18 18:21:27 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('postulante/admi...')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii/prueba/postulante/admin
HTTP_REFERER=http://localhost:7777/yii/prueba/postulante/index
---
2015/07/18 18:46:03 [error] [php] Undefined variable: model (C:\xampp\htdocs\yii\prueba\protected\views\site\login.php:28)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php(32): SiteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/site/index
in C:\xampp\htdocs\yii\prueba\protected\views\site\login.php (28)
in C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php (32)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 18:46:06 [error] [php] Undefined variable: model (C:\xampp\htdocs\yii\prueba\protected\views\site\login.php:28)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php(32): SiteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/site/index
in C:\xampp\htdocs\yii\prueba\protected\views\site\login.php (28)
in C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php (32)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 19:42:55 [error] [exception.CException] exception 'CException' with message 'Property "CNumberValidator.0" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\validators\CValidator.php(177): CComponent->__set('0', 'unique')
#1 C:\xampp\htdocs\yii\framework\base\CModel.php(287): CValidator::createValidator('numerical', Object(Postulante), 'Cedula, idEstad...', Array)
#2 C:\xampp\htdocs\yii\framework\base\CModel.php(260): CModel->createValidators()
#3 C:\xampp\htdocs\yii\framework\base\CModel.php(304): CModel->getValidators('Nombre')
#4 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1513): CModel->isAttributeRequired('Nombre')
#5 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(600): CHtml::activeLabelEx(Object(Postulante), 'Nombre', Array)
#6 C:\xampp\htdocs\yii\prueba\protected\views\postulante\_form.php(23): CActiveForm->labelEx(Object(Postulante), 'Nombre')
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#8 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#10 C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php(18): CController->renderPartial('_form', Array)
#11 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#12 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#13 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#14 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#15 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(79): CController->render('create', Array)
#16 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#18 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#19 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#20 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#21 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#22 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#23 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#24 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#25 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#26 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('postulante/crea...')
#27 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#28 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#29 {main}
REQUEST_URI=/yii/prueba/postulante/create
HTTP_REFERER=http://localhost:7777/yii/prueba/postulante/create
---
2015/07/18 20:11:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "site".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('site')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('site')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('postulante/site...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii/prueba/postulante/site/login
HTTP_REFERER=http://localhost:7777/yii/prueba/postulante/create
---
2015/07/18 20:14:23 [error] [exception.CException] exception 'CException' with message 'PostulanteController cannot find the requested view "login".' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('login', Array, true)
#1 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(76): CController->render('login', Array)
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('postulante/crea...')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#14 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yii/prueba/postulante/create
HTTP_REFERER=http://localhost:7777/yii/prueba/postulante/create
---
2015/07/18 20:14:50 [error] [exception.CException] exception 'CException' with message 'PostulanteController cannot find the requested view "site/login".' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('site/login', Array, true)
#1 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(76): CController->render('site/login', Array)
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('postulante/crea...')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#14 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yii/prueba/postulante/create
HTTP_REFERER=http://localhost:7777/yii/prueba/postulante/create
---
2015/07/18 20:32:34 [error] [exception.CException] exception 'CException' with message 'PostulanteController cannot find the requested view "site/login".' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('site/login', Array, true)
#1 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(76): CController->render('site/login', Array)
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('postulante/crea...')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#14 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yii/prueba/postulante/create
HTTP_REFERER=http://localhost:7777/yii/prueba/postulante/create
---
2015/07/18 20:35:59 [error] [exception.CException] exception 'CException' with message 'PostulanteController cannot find the requested view "application.site.login".' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(77): CController->renderPartial('application.sit...', Array)
#1 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('postulante/crea...')
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#13 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/yii/prueba/postulante/create
HTTP_REFERER=http://localhost:7777/yii/prueba/postulante/create
---
2015/07/18 20:36:51 [error] [exception.CException] exception 'CException' with message 'Property "Postulante.username" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('username')
#1 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2728): CActiveRecord->__get('username')
#2 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2451): CHtml::resolveValue(Object(Postulante), 'username')
#3 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1533): CHtml::activeInputField('text', Object(Postulante), 'username', Array)
#4 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(792): CHtml::activeTextField(Object(Postulante), 'username', Array)
#5 C:\xampp\htdocs\yii\prueba\protected\views\site\login.php(29): CActiveForm->textField(Object(Postulante), 'username')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#9 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(77): CController->renderPartial('application.vie...', Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#17 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#18 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#20 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('postulante/crea...')
#21 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#22 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#23 {main}
REQUEST_URI=/yii/prueba/postulante/create
HTTP_REFERER=http://localhost:7777/yii/prueba/postulante/create
---
2015/07/18 20:37:52 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "login".' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('login', Array, true)
#1 C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php(98): CController->render('login', Array)
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/prueba/site/login
HTTP_REFERER=http://localhost:7777/yii/prueba/site/contact
---
2015/07/18 20:39:02 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "login".' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('login', Array, true)
#1 C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php(98): CController->render('login', Array)
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/prueba/site/login
HTTP_REFERER=http://localhost:7777/yii/prueba/site/contact
---
2015/07/18 20:39:12 [error] [php] Object of class PostulanteController could not be converted to string (C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php:74)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (74)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 20:40:01 [error] [php] Object of class PostulanteController could not be converted to string (C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php:74)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (74)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 20:40:56 [error] [php] Object of class PostulanteController could not be converted to string (C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php:74)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (74)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 20:51:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "vistaExito".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('vistaExito')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('vistaExito')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('postulante/vist...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii/prueba/postulante/vistaExito
HTTP_REFERER=http://localhost:7777/yii/prueba/postulante/create
---
2015/07/18 20:51:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "vistaExito".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('vistaExito')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('vistaExito')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('postulante/vist...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii/prueba/postulante/vistaExito
HTTP_REFERER=http://localhost:7777/yii/prueba/postulante/create
---
2015/07/18 21:11:01 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create?%24mensaje=Te+has+registrado+correctamente%21
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:11:06 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:11:15 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:11:17 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:11:18 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:11:19 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:11:19 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:11:34 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:11:35 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:11:36 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:11:36 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:12:03 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:12:04 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:12:04 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:12:04 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:12:06 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:13:39 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:13:40 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:13:40 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:13:40 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:15:11 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:15:13 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:15:14 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:15:14 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:15:15 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:15:24 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:15:25 [error] [php] Undefined variable: mensaje (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(81): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (19)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (81)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:18:24 [error] [php] Undefined variable: model (C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php:18)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): PostulanteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): PostulanteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php(77): PostulanteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostulanteController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): PostulanteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): PostulanteController->filterAccessControl()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): PostulanteController->runActionWithFilters()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): PostulanteController->run()
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/postulante/create
in C:\xampp\htdocs\yii\prueba\protected\views\postulante\create.php (18)
in C:\xampp\htdocs\yii\prueba\protected\controllers\PostulanteController.php (77)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 21:24:52 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('postulante/admi...')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii/prueba/postulante/admin
HTTP_REFERER=http://localhost:7777/yii/prueba/site/login
---
2015/07/18 23:45:22 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in C:\xampppp\htdocs\yii\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampppp\htdocs\yii\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampppp\htdocs\yii\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampppp\htdocs\yii\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampppp\htdocs\yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampppp\htdocs\yii\prueba\protected\components\UserIdentity.php(20): CModule->__get('db')
#5 C:\xampppp\htdocs\yii\prueba\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#6 C:\xampppp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#7 C:\xampppp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#8 C:\xampppp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#9 C:\xampppp\htdocs\yii\prueba\protected\controllers\SiteController.php(94): CModel->validate()
#10 C:\xampppp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#11 C:\xampppp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampppp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampppp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampppp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#15 C:\xampppp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#16 C:\xampppp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#17 C:\xampppp\htdocs\yii\prueba\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/yii/prueba/site/login
HTTP_REFERER=http://localhost/yii/prueba/site/login
---
2015/07/18 23:45:42 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in C:\xampppp\htdocs\yii\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampppp\htdocs\yii\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampppp\htdocs\yii\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampppp\htdocs\yii\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampppp\htdocs\yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampppp\htdocs\yii\prueba\protected\components\UserIdentity.php(20): CModule->__get('db')
#5 C:\xampppp\htdocs\yii\prueba\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#6 C:\xampppp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#7 C:\xampppp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#8 C:\xampppp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#9 C:\xampppp\htdocs\yii\prueba\protected\controllers\SiteController.php(94): CModel->validate()
#10 C:\xampppp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#11 C:\xampppp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampppp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampppp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampppp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#15 C:\xampppp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#16 C:\xampppp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#17 C:\xampppp\htdocs\yii\prueba\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/yii/prueba/site/login
HTTP_REFERER=http://localhost/yii/prueba/site/login
---
